docs/docs/releases/0.1.4.md

```diff
@@ -143,3 +141,210 @@ The table sidebar features a new "Cell" tab to show the content of cells, simpli
 - We improved our process for generating release notes. _([#3427](https://github.com/mathesar-foundation/mathesar/pull/3427))_
 - We resolved some merge conflicts after finalizing our previous release. _([#3190](https://github.com/mathesar-foundation/mathesar/pull/3190))_
 
+## Upgrading to 0.1.4 {:#upgrading}
```

Comment:
One non-blocking nitpick with the `Upgrading to {version}` sections: The previous language used was "upgrade to" which I think is preferable, specifically in the context of searching the docs for the term "upgrade" in the searchbar, which I attempted to do in order to find these changes for review. 

Or, perhaps the "See the [upgrading](http://localhost:9000/releases/0.1.7/#upgrading) section below for instructions on how to upgrade to this version." text could say "See the [upgrade guide](http://localhost:9000/releases/0.1.7/#upgrading) below for instructions on how to upgrade to this version."

That way the terms "upgrade" and "upgrading" are both represented on the page. 

Overall though I think the idea to colocate the upgrade guide with the release information is excellent though.

Reply:
Good point. I added a separate "Upgrade Mathesar" page in 84602157c35a2b5838ffed5c58869578ba87a898 which I hope will address this.